docs/cash-savings/Interest.md

```diff
@@ -0,0 +1,31 @@
+---
+tags: [Cash Savings]
+---
+
+# Interest
+
+## How interest is paid
+
+Interest is paid on the 1st of each month on an accrual basis. This means that interest is calculated (accrued) daily, but paid monthly. 
+
+## Interest rates
+
+Interest rates can be configured per product, split between the investor and the tenant, up to a maximum of the current [Bank of England base rate](https://www.bankofengland.co.uk/monetary-policy/the-interest-rate-bank-rate).
```

Comment:
Should we tweak this to indicate tenants should speak to TMT to configure interest rate?